src/azure-cli/azure/cli/command_modules/appconfig/custom.py

```diff
@@ -84,7 +123,8 @@ def update_configstore(cmd,
                        encryption_key_version=None,
                        identity_client_id=None,
                        enable_public_network=None,
-                       disable_local_auth=None):
+                       disable_local_auth=None,
+                       enable_purge_protection=False):
```

Comment:
>False [](http://example.com/codeflow?start=47&length=5)

I think it makes sense to keep it 'None" here. What if someone enables purge protection, and then runs update command to update something else? We will still pass False to the server and the operation might fail because we are trying to turn off purge protection.

Reply:
Good catch!